src/upload.d

```diff
@@ -268,6 +268,15 @@ struct UploadSession
 		}
 	}
 	
+	string getSessionFilePath() {
```

Comment:
Should you name this method something like "getSessionLocalPath"?  
`sessionFilePath` is an existing member attribute, took me some time to understand that this new `getSessionFilePath()` doesn't return `sessionFilePath`

Reply:
Resolved by https://github.com/abraunegg/onedrive/pull/1399/commits/9cdf81b074715c9f07a5be92d99d7bf5c22cc324